packages/flutter_driver/lib/src/common/find.dart

```diff
@@ -230,7 +230,7 @@ class ByText extends SerializableFinder {
 /// A Flutter Driver finder that finds widgets by `ValueKey`.
 class ByValueKey extends SerializableFinder {
   /// Creates a finder given the key value.
-  ByValueKey(this.keyValue)
+  ByValueKey(this.keyValue, {this.skipOffstage = true})
```

Comment:
Can you please add a unit test for this feature to flutter_driver?

Reply:
I actually meant to revert that.  Reverted.